ArcGISRuntimeSDKQt_CppSamples/Routing/RouteAroundBarriers/RouteAroundBarriers.qml

```diff
@@ -139,23 +138,28 @@ Item {
                 Layout.margins: 3
                 color: "lightgrey"
 
-                ListView {
-                    id: directionsView
-                    anchors {
-                        fill: parent
-                        margins: 5
-                    }
-                    header: Component {
-                        Text {
-                            height: 40
-                            text: "Directions:"
-                            font.pixelSize: 22
+                ScrollView {
+                    width: parent.width
```

Comment:
does `anchors.fill: parent` not work on the ScrollView?

Reply:
it does work, I will make this change